src/ProcessManager/WebServerReadinessProbeTrait.php

```diff
@@ -55,7 +55,7 @@ public function waitUntilReady(Process $process, string $url, string $service, b
 
             if (Process::STATUS_STARTED !== $status) {
                 if (microtime(true) - $start >= $timeout) {
-                    throw new \RuntimeException("Could not start $service (or it crashed) after $timeout seconds.");
+                    throw new \RuntimeException("Could not start ${service} (or it crashed) after ${timeout} seconds.");
```

Comment:
it could be better to use `{$service}`, `{$timeout}`, and `{$message}` everywhere for [future compatibility](https://wiki.php.net/rfc/deprecate_dollar_brace_string_interpolation)